### PR TITLE
Issue8#change username

### LINKED ISSUE #8: A user should be able to change it's username, but not its password
Note that if the new username already exists in the system, the user will need to pick another username.

### PR DESCRIPTION
**Pull request by e-letos-levi9-com**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -14,5 +14,16 @@ libraryDependencies ++= Seq(
...
+"org.tpolecat" %% "doobie-core"      % "0.12.1",
+
+// And add any of these as needed
+"org.tpolecat" %% "doobie-h2"        % "0.12.1",          // H2 driver 1.4.200 + type mappings.
+"org.tpolecat" %% "doobie-hikari"    % "0.12.1",          // HikariCP transactor.
+"org.tpolecat" %% "doobie-postgres"  % "0.12.1",          // Postgres driver 42.2.19 + type mappings.
+"org.tpolecat" %% "doobie-quill"     % "0.12.1",          // Support for Quill 3.6.1
+"org.tpolecat" %% "doobie-specs2"    % "0.12.1" % "test", // Specs2 support for typechecking statements.
+"org.tpolecat" %% "doobie-scalatest" % "0.12.1" % "test"  // ScalaTest support for typechecking statements.
 )
```
Comment (arcilli) [inline build.sbt]: missing newline at eof

File: src/main/scala/DataBaseConn.scala [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+  import scala.concurrent.ExecutionContext
+
+  implicit val cs = IO.contextShift(ExecutionContext.global)
+
+  val xa = Transactor.fromDriverManager[IO](
+    "org.postgresql.Driver", "jdbc:postgresql:browserhistory", "postgres", ""
+  )
+
+  def findUserByName(username: String)  = {
```
Comment (arcilli) [inline src/main/scala/DataBaseConn.scala]: Can we do an interface containing those methods and the actual implementation to be in an implementation of that interface?

File: src/main/scala/DataBaseConn.scala [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  def updateUsername(usernameFirst: String, usernameSecond: String) = {
+    sql"update users set username = $usernameSecond where username=$usernameFirst"
+      .update
+      .run
+      .transact(xa)
+      .unsafeRunSync()
+  }
+
+  def findUser(username: String, password: String) ={
+    sql"select * from users where username = $username and password = $password"
```
Comment (arcilli) [inline src/main/scala/DataBaseConn.scala]: What about SQL injection?

**Review by arcilli — COMMENTED** [review]
Nice work. I left some comments.

**Review by dia28 — APPROVED** [review]

**Review by g-antoci-levi9-com — APPROVED** [review]

**Pull request merged**